src/bilder/images/concourse/templates/vector.yaml

```diff
@@ -19,28 +25,61 @@ transforms:
     source: |
       event, err = parse_json(.message)
       if event != null {
-        . = merge(., event)
+        .,err = merge(., event)
         .@timestamp = .timestamp
         .labels = ["concourse", "${ENVIRONMENT}"]
         .environment = "${ENVIRONMENT}"
       }
 
+  add_labels_to_metrics:
+    type: remap
+    inputs:
+    - '*metrics'
+    source: |
+      .tags.environment = "${ENVIRONMENT}"
+      .tags.application = "concourse"
+      .tags.service = "concourse"
+      .tags.hostname = "${HOSTNAME}"
+
   concourse_enriched_logs:
     type: aws_ec2_metadata
     inputs:
     - concourse_parsed_message
     namespace: ec2
 
 sinks:
-  elasticsearch_logs:
-    type: elasticsearch
+  grafana_loki_logs:
     inputs:
     - concourse_enriched_logs
-    endpoint: http://logging-elasticsearch.query.consul:9200
-    index: logs-${ENVIRONMENT}-concourse-%Y.%W
-    healthcheck: false
+    type: loki
+    auth:
+      strategy: basic
+      password: ${GRAFANA_CLOUD_API_KEY}
+      user: "${GRAFANA_CLOUD_LOKI_API_USER}"
+    endpoint: https://logs-prod-us-central1.grafana.net
+    encoding:
+      codec: json
+    labels:
+      environment: ${ENVIRONMENT}
+      application: concourse
+      service: concourse
+      hostname: ${HOSTNAME}
```

Comment:
Let's skip on the hostname label for logs so that it doesn't contribute too much to explosion of log streams.